apps/pipeline/upstream/base/installs/multi-user/istio-authorization-config.yaml

```diff
@@ -32,10 +32,9 @@ spec:
         - cluster.local/ns/kubeflow/sa/ml-pipeline-scheduledworkflow
         - cluster.local/ns/kubeflow/sa/ml-pipeline-viewer-crd-service-account
         - cluster.local/ns/kubeflow/sa/kubeflow-pipelines-cache
-  # For user workloads, which cannot user http headers for authentication
-  - when:
-    - key: request.headers[kubeflow-userid]
-      notValues: ['*']
+  - from:
+    - source:
+        requestPrincipals: ["*"]  # allow access by any trusted principal
```

Comment:
This is the important thing to approve. The rest is rather straightforward.

Reply:
@kimwnasptd wants it in https://github.com/kubeflow/manifests/issues/2747#issuecomment-2151541954 so it should be approved by him.

Reply:
Agree with the changes, although these should be reflected in the `kubeflow/pipelines` project and not directly in the manifests repo

Reply:
Yes, can you track that with them? Otherwise it will break in the next synchronization.